src/api/app/controllers/webui2/package_controller.rb

```diff
@@ -25,4 +25,21 @@ def webui2_statistics
                                                       repository: @repository,
                                                       architecture: params[:arch]).results
   end
+
+  def webui2_branch_diff_info
+    cleanup_source = @project.branch? # We should remove the package if this request is a branch
+    linked_package = @package.backend_package.links_to
+    if linked_package
+      target_project = linked_package.project.name
+      target_package = linked_package.name
+      description = @package.commit_message(target_project, target_package)
+    end
+
+    render json: {
+      'targetProject': defined?(target_project) ? target_project : '',
```

Comment:
is that better than `linked_package ? linked_package.project.name : ' '` ?

Reply:
Yes, I need the `target_project` and `target_package` for the `description`, so there is no reason to not use them there...